comet/output.py

```diff
@@ -1,13 +1,142 @@
-#!/usr/bin/python
-
-# Load required modules
+# output comet results and website
 import sys, os, json, math, argparse, time, networkx as nx, tempfile
-import comet as C
 from collections import defaultdict
 import numpy as np
 from itertools import combinations as combos
 from math import factorial
 
+def ensure_dir(path):
+	try: 
+		os.makedirs(path)
+	except OSError:
+		if not os.path.isdir(path):
+			raise
+
+def iter_num (prefix, numIters, ks, acc):
+
+	if numIters >= 1e9: iterations = "%sB" % (numIters / int(1e9))
+	elif numIters >= 1e6: iterations = "%sM" % (numIters / int(1e6))
+	elif numIters >= 1e3: iterations = "%sK" % (numIters / int(1e3))
+	else: iterations = "%s" % numIters
+
+	prefix += ".k%s.%s.%s" % ("".join(map(str, ks)), iterations, acc)
+	return prefix
+
+def output_comet(args, mutations, results, collections, ks, runNum, maxPermutedWeight, permutedN):
+	weight_func_mapping = {0: 'E', 1:'E', 2:'B', 3:'P'}
```

Comment:
These should use the constants from `comet/constants.py`. I think you can even use `weightFunctionNames` directly.